### PR TITLE
Compose things

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

**Comment by mdellweg** [comment]
Originally, i wanted to make use of secrets in the compose files. But i do not see that i will get to it.
Please consider the commits individually.

File: .github/workflows/pulp_images.yml [hunk 1]
```diff
@@ -240,11 +240,11 @@ jobs:
         run: |
           if [ "${{ matrix.app.s6 }}" == "true" ]; then
             # Reuse the folders from the s6 mode tests
-            FILE="docker-compose.folders.yml"
+            FILE="compose.folders.yml"
             # We'll pull the web image from a registry since we didn't build it.
             WEB_TAG="${APP_BRANCH}"
           else
-            FILE="docker-compose.yml"
+            FILE="compose.yml"
```
Comment (mikedep333) [inline .github/workflows/pulp_images.yml:273]: I was looking at the compose spec, and I see that docker-compose now prefers compose.yaml or compose.yml . Nice!

File: images/compose/compose.folders.yaml [hunk 2]
```diff
@@ -23,9 +23,10 @@ services:
     image: "pulp/pulp-web:latest"
     command: ['/usr/bin/nginx.sh']
     depends_on:
-      - postgres
-      - pulp_api
-      - pulp_content
+      postgres:
+        condition: service_healthy
```
Comment (mikedep333) [inline images/compose/compose.folders.yaml]: Nice feature!

File: images/compose/compose.yaml [hunk 3]
```diff
@@ -14,6 +14,11 @@ services:
       - "pg_data:/var/lib/postgresql"
       - "./assets/postgres/passwd:/etc/passwd:Z"
     restart: always
+    healthcheck:
```
Comment (mikedep333) [inline images/compose/compose.yaml]: ❤️

**Review by mikedep333 — APPROVED** [review]

**Pull request merged**